Userland/Libraries/LibWeb/CSS/Parser/MediaParsing.cpp

```diff
@@ -32,6 +32,13 @@ Vector<NonnullRefPtr<MediaQuery>> Parser::parse_a_media_query_list(TokenStream<T
     AK::Vector<NonnullRefPtr<MediaQuery>> media_queries;
     for (auto& media_query_parts : comma_separated_lists) {
         auto stream = TokenStream(media_query_parts);
+
+        // AD-HOC: Ignore whitespace-only queries
+        // to make `@media {..}` equivalent to `@media all {..}`
+        stream.skip_whitespace();
+        if (!stream.has_next_token())
+            continue;
```

Comment:
This seems subtly wrong. It'll parse `@media ,,,,, {}` as an empty media query list instead of as an invalid one.

Can we check for whitespace-only before dividing it up by commas instead?